### ISSUE TITLE
Cannot find type 'RCTEventDispatcher' in scope

### ISSUE THREAD

**Issue by stereodenis** [TheWidlarzGroup/react-native-video#3152]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-video@6.0.0-alpha.6` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-video/ios/Video/RCTVideo-Bridging-Header.h b/node_modules/react-native-video/ios/Video/RCTVideo-Bridging-Header.h
index 77815e4..586eec1 100644
--- a/node_modules/react-native-video/ios/Video/RCTVideo-Bridging-Header.h
+++ b/node_modules/react-native-video/ios/Video/RCTVideo-Bridging-Header.h
@@ -1,5 +1,6 @@
 #import <React/RCTViewManager.h>
 #import "RCTVideoSwiftLog.h"
+#import "RCTEventDispatcher.h"
 
 #if __has_include(<react-native-video/RCTVideoCache.h>)
 #import "RCTVideoCache.h"
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by jkastensmidt** [comment]
+1 this fixes the issue

**Comment by freeboub** [comment]
I integrated this change, will be available in 6.0.0-alpha.7 (or beta.1 !)
